twindle-cli/pdf/create-pdf.js

```diff
@@ -3,19 +3,19 @@ const fs = require("fs");
 
 // Creates a pdf document from htmlContent and saves it to outputPath
 async function createPdf(outputPath, htmlContent) {
-	// launchs a puppeteer browser instance and opens a new page
-	const browser = await puppeteer.launch();
-	const page = await browser.newPage();
+  // launchs a puppeteer browser instance and opens a new page
+  const browser = await puppeteer.launch({ args: ["--no-sandbox"] });
```

Comment:
This was required for my WSL environment and it is also required when this script will be turned over to a Serverless environment

Reply:
Yeah very True, also it is needed to run the code in linux servers

Reply:
When I applied these changes locally the PDF file was not being generated, though no errors appeared.

Reply:
no-sandbox doesn't seem to be working on Windows.